default_coverage_report.toml

```diff
@@ -0,0 +1,11 @@
+[tool.coverage.report]
```

Comment:
Is there any particular reason for creating a new file instead of modifying `pyproject.toml`?

Reply:
As I mentioned above, I didn't know the impact of changing it and happy to merge it as one file.

Reply:
Merged the config into `pyproject.toml` via 8bb4db990ce34aa57d90b0de66214427653278e6